run_cgw.sh

```diff
@@ -40,12 +42,12 @@ echo "KAFKA: $KAFKA_IP:$KAFKA_PORT"
 echo "DB: $DB_IP:$DB_PORT"
 echo "REDIS_DB: $REDIS_DB_IP:$REDIS_DB_PORT"
 
-# TBD: docker-based run
-# TBD 2: add num of thread cfg (start argument)
-output/bin/ucentral-cgw -w 4 -c $ID \
+# TBD: add num of thread cfg (start argument)
+docker run -d -t --network=host --name $2 $1 ucentral-cgw \
 	--grpc-ip $GRPC_IP --grpc-port $GRPC_PORT \
 	--wss-ip $WSS_LOCAL_IP --wss-port $WSS_LOCAL_PORT \
 	--kafka-ip $KAFKA_IP --kafka-port $KAFKA_PORT \
 	--db-ip $DB_IP --db-port $DB_PORT \
+	--db-username $DB_USR --db-password $DB_PASS \
 	--redis-db-ip $REDIS_DB_IP --redis-db-port $REDIS_DB_PORT \
```

Comment:
I would suggest use environment variables to pass to container, i.e. use `-e KAFKA_IP=$KAFKA_IP`, and use environment variables inside the application. We can discuss it.

Reply:
i think for purposes of additional services  kafka, redis, pgsql the .env style might be more appropriate

Reply:
We can change this to use run_cgw script inside docker.. this way we can utilize env vars without coding the CGW itself to use variables and stick to the args;

Reply:
@Cahb I would prefer we use environment variables within the application, and not command line arguments.   It is a security thing: anyone you can get running process information can look at command line args and see the private info.   Best practices for security purposes is to pass this in via a environment variables, or a second backup would be a config file with the right permissions read by the app.   I think the environment variable option is better in our case.

Reply:
@phwhite, @i-chvets @outbackdingo are you guys OK with changin args -> env variables inside application as a separate change? We'll add this to our backlog, as it's not a functional change and rather replacement of existing (arguments-approach) functionality

Reply:
> @phwhite, @i-chvets @outbackdingo are you guys OK with changin args -> env variables inside application as a separate change? We'll add this to our backlog, as it's not a functional change and rather replacement of existing (arguments-approach) functionality

Sure, I'm fine with this